platform/mds/mds-web/src/main/java/org/motechproject/mds/web/controller/MdsController.java

```diff
@@ -27,7 +29,9 @@
     @ResponseBody
     public String handleMdsException(final MdsException exception) throws IOException {
         LOGGER.error("Error: " + exception.getMessage(), exception);
-
+        if (exception instanceof ObjectCreateException || exception instanceof ObjectUpdateException) {
```

Comment:
Who told you to do this? We still want the localized message for these exceptions